features/facebookImport/src/static/manifest.json

```diff
@@ -6,6 +6,8 @@
     "primaryColor": "#0f1938",
     "links": {
         "data-download": "https://www.facebook.com/dyi",
+        "example-data-download-de": "https://polypoly.coop/de-de/polypod/example-facebook-data",
+        "example-data-download-en": "https://polypoly.coop/en-de/polypod/example-facebook-data",
```

Comment:
Drop the language suffix and just add the German link with the same key below under `translations` - the polyPreview feature shows how to do it.

Reply:
Done! Missed that the first time I looked at polyPreview